wandb/sdk/service/streams.py

```diff
@@ -0,0 +1,305 @@
+#!/usr/bin/env python
+"""wandb grpc server.
+
+- WandbServer:
+- StreamMux:
+- StreamRecord:
+- WandbServicer:
+
+"""
+
+import multiprocessing
+import queue
+import threading
+from threading import Event
+import time
+from typing import Any, Callable, Dict, List, Optional, Union
+from typing import TYPE_CHECKING
+
+import wandb
+from wandb.proto import wandb_internal_pb2 as pb
+from wandb.proto import wandb_server_pb2 as spb
+
+from ..interface.interface_relay import InterfaceRelay
+
+
+if TYPE_CHECKING:
+    from google.protobuf.internal.containers import MessageMap
+
+    class GrpcServerType(object):
```

Comment:
probably can remove this one, doesn't seem to be used

Reply:
removed